server/routes/profile.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const router = express.Router();
+const {createProfile, getProfile, getAllProfiles, updateprofile} = require('../controllers/profile');
+
+router.route('/createprofile').post(createProfile);
+router.route('/profile/:id').get(getProfile);
+router.route('/getallprofiles').get(getAllProfiles);
+router.route('/updateprofile/:id').put(updateprofile)
```

Comment:
For better readability, i would follow camelCases for routes for all of these

```suggestion
router.route('/updateProfile/:id').put(updateProfile)
```